lib.sh

```diff
@@ -850,6 +850,27 @@ to validate them with the $f method. We have exhausted all the methods. Please c
 done
 }
 
+# Let the user choose to setup a specific app with either deSEC, or regular TLS.
+# desec_app_tls_menu "DESEC collabora_docker_desec" "APP collabora_docker"
+desec_app_tls_menu() {
+choice=$(whiptail --title "$TITLE" --menu \
+"Choose TLS setup. Please note, to run the deSEC option, deSEC needs to be configured and setup already.\n
+$MENU_GUIDE\n\n$RUN_LATER_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+"deSEC TLS setup" "(If you configured deSEC already. Works with custom port.)" \
+"Regular TLS setup" "(If deSEC isn't installed, setup normal TLS)" 3>&1 1>&2 2>&3)
```

Comment:
Can we make Regular TLS setup the default if you want to keep this pre-menu?

Reply:
Sure